exporter/stats/stackdriver/stackdriver.go

```diff
@@ -187,14 +187,14 @@ func (e *Exporter) makeReq(vds []*stats.ViewData) *monitoringpb.CreateTimeSeries
 }
 
 func (e *Exporter) createMeasure(ctx context.Context, vd *stats.ViewData) error {
-	e.measuresMu.Lock()
-	defer e.measuresMu.Unlock()
+	e.viewMu.Lock()
+	defer e.viewMu.Unlock()
 
 	m := vd.View.Measure()
 	agg := vd.View.Aggregation()
 	window := vd.View.Window()
```

Comment:
name := d.View.Name()

Then use it elsewhere.

Reply:
Done. I named it `viewName`, to avoid confusion with `metricName`.